tests/back/Pim/Enrichment/EndToEnd/Product/ProductModel/ExternalApi/AbstractProductModelTestCase.php

```diff
@@ -125,6 +126,21 @@ protected function getStandardizedProductModels()
         return $standardizedProductModels;
     }
 
+    /**
+     * @param array  $expectedProductModel normalized data of the product model that should be created
+     * @param string $identifier           identifier of the product that should be created
+     */
+    protected function assertSameProductModels(array $expectedProductModel, $identifier)
```

Comment:
Maybe rename $identifier to $code even if we keep call to findOneByIdentifier ?

```suggestion
     * @param array  $expectedProductModel normalized data of the product model that should be 
     * @param string $code           code of the product model that should be compared
     */
    protected function assertSameProductModels(array $expectedProductModel, string $identifier): void
```